### PR TITLE
Add Affiliation scaffold

### PR DESCRIPTION
**Pull request by yhelen**

### CODE REVIEW

File: app/models/student.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ class Student < ApplicationRecord
 
   has_and_belongs_to_many :parents
   has_and_belongs_to_many :trips
+  has_and_belongs_to_many :affiliations
```
Comment (justinkim) [inline app/models/student.rb]: alpha order if you don't mind please

File: db/migrate/20180223152923_create_affiliations.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateAffiliations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :affiliations do |t|
+      t.string :name, null: false
```
Comment (justinkim) [inline db/migrate/20180223152923_create_affiliations.rb]: Can we get an index on this field

File: db/schema.rb [hunk 3]
```diff
@@ -138,32 +151,8 @@
...
-    t.index ["is_newbie"], name: "index_students_on_is_newbie"
-    t.index ["last_name"], name: "index_students_on_last_name"
-    t.index ["osis"], name: "index_students_on_osis", unique: true
-    t.index ["phone"], name: "index_students_on_phone"
-    t.index ["preferred_name"], name: "index_students_on_preferred_name"
-    t.index ["team_id"], name: "index_students_on_team_id"
-    t.index ["user_id"], name: "index_students_on_user_id"
-  end
+# Could not dump table "students" because of following StandardError
+#   Unknown type 'gender' for column 'string'
```
Comment (justinkim) [inline db/schema.rb]: Some sort of problem here. You need to reset the db and re-migrate on this branch without the gender changes

File: test/fixtures/affiliations.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+engineering:
```
Comment (justinkim) [inline test/fixtures/affiliations.yml]: mechanical please

File: test/fixtures/affiliations.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  name: "Engineering"
+
+se:
+  name: "Software Engineering"
+
+marketing:
+  name: "Marketing"
+
+driveteam:
+  name: "Driveteam"
```
Comment (justinkim) [inline test/fixtures/affiliations.yml]: Drive Team please

File: test/fixtures/affiliations.yml (hunk 4 above)
Comment (justinkim) [reply test/fixtures/affiliations.yml]: You can say `me` like `se`

File: app/models/student.rb (hunk 1 above)
Comment (yhelen) [reply app/models/student.rb]: does that mean alphabetical

**Pull request merged**